client/src/lib/templates/LandingTemplate/LandingTemplate.svelte

```diff
@@ -0,0 +1,69 @@
+<script>
```

Comment:
? (On devrait le rajouter plus tard si jamais on ajoute du JS ici, de toute façon...)

```suggestion
<script lang="ts">
```